source/security/index.html.md.erb

```diff
@@ -17,6 +17,8 @@ Please do not disclose the suspected breach publicly until it has been fixed.
 
 GOV.UK Pay allows you to create as many API keys as you want.
 
+This only applies if you [integrate with the GOV.UK Pay API](/integrate_with_govuk_pay/#integrate-with-the-gov-uk-pay-api).
```

Comment:
A user can technically create an API key on a payment links service but they don't need to. I wonder if we just leave this section as is?

Reply:
ok will remove